scene/main/viewport.cpp

```diff
@@ -118,7 +118,6 @@ Size2 ViewportTexture::get_size() const {
 }
 
 RID ViewportTexture::get_rid() const {
-	//ERR_FAIL_COND_V_MSG(!vp, RID(), "Viewport Texture must be set to use it.");
```

Comment:
Commented out 4 years ago in https://github.com/godotengine/godot/commit/d438ac0aeda8509e5961296a5950c2755ba8a966